### PR TITLE
Add zombie cow and zombie deer

### PR DESCRIPTION
**Pull request by LyleSY**
#### Summary

SUMMARY: Content "Add zombie cow and zombie deer"

#### Purpose of change

Add some monster variety in farm and wilderness areas, make world feel more consistent

#### Describe the solution

Applies standard zombie template to cow and deer, can still get filthy cowbells. Cow and deer will turn after death. Spawns zombie deer in woods on a timer

#### Describe alternatives you've considered

Coyote and fox but they're too small. Mass of zombie foxes? Giant fox? Zombie upgrades with fox heads? Coyote same? Sheep.

#### Testing

Loads, spawns, no errors
image

#### Additional context

Part of general desire for animals to feel like part of the world, and for stuff to get weirder and harder as time passes

### CODE REVIEW

File: data/json/monsters/zed-animal.json [hunk 1]
```diff
@@ -383,6 +383,69 @@
...
     "dodge": 0
   },
+  {
+    "id": "mon_zow",
+    "type": "MONSTER",
+    "name": { "str": "zombie cow" },
+    "description": "Once a placid cow, this leathery horror stumbles and shudders, its pulsing black eyes scanning for prey beneath wicked horns.  It is big and heavy and murderous.",
+    "looks_like": "mon_cow",
+    "default_faction": "zombie",
+    "bodytype": "dog",
```
Comment (Night-Pryanik) [inline data/json/monsters/zed-animal.json:393]: I guess it should have cow bodytype?

Comment (LyleSY) [reply data/json/monsters/zed-animal.json:393]: Not implemented yet to my knowledge, though I don't fully understand how bodytype is used

Comment (MatiasFito) [reply data/json/monsters/zed-animal.json:393]: Bodytype is used in some calculations, for grabs and snares when fighting monsters (and monsters fighting each other too). Like in this example:
Screenshot from 2020-07-13 09-22-50

Also it is used to determine the kind of armor friendly animals can wear, as it is today in Stable you can equip your cow (that is also "dog" bodytype) with a K-9 Kevlar harness.
Screenshot from 2020-07-13 09-26-57

Cow (or cattle) bodytype is not in the game as @LyleSY said, so for the purpose of having Zows i believe it's perfectly fine to replicate that value from the Original Cow. Looking forward to more zed fauna, i love playing wayward size 1 cities playthroughs.

Comment (Night-Pryanik) [reply data/json/monsters/zed-animal.json:393]: If I'm reading it right, maximum monster volume for this harness is 350 liters, while cows have 680 liters.

**Pull request merged**